docker/test

```diff
@@ -61,28 +61,28 @@ sleep 3
 
 # Run a basic status check; we don't particularly care about the output itself,
 # so essentially we're just checking for the exit code.
-docker compose ${compose_files} run vast status > /dev/null
+docker compose run vast status > /dev/null
 
 # Ingest some test data.
 gunzip -c ../../vast/integration/data/zeek/*.log.gz |
-  docker compose ${compose_files} run --no-TTY vast import --blocking zeek
-docker compose ${compose_files} run --no-TTY vast import --blocking suricata \
+  docker compose run --no-TTY vast import --blocking zeek
+docker compose run --no-TTY vast import --blocking suricata \
   < ../../vast/integration/data/suricata/eve.json
 
 # Check whether we imported the right amount of data.
-num_imported="$(docker compose ${compose_files} run --interactive=false vast count)"
+num_imported="$(docker compose run --interactive=false vast count)"
 num_imported_expected="18493"
 [[ "${num_imported}" == "${num_imported_expected}" ]]
 
 # Print logs for easier debugging in case something went wrong.
-docker compose ${compose_files} ps
-docker compose ${compose_files} logs
+docker compose ps
+docker compose logs
 
 # Assert that all containers that exited have exited cleanly.
 exited_all="$(docker ps --all --filter "name=${COMPOSE_PROJECT_NAME}" --filter 'status=exited' -q | sort -u)"
 exited_cleanly="$(docker ps --all --filter "name=${COMPOSE_PROJECT_NAME}" --filter 'exited=0' -q | sort -u)"
```

Comment:
Isn't `docker compose ps --all` an alternative to `docker ps --all --filter "name=${COMPOSE_PROJECT_NAME}"`. That would spare us the dance around the container name.

Reply:
TIL that `docker compose ps` exists. That does make a lot of things a lot easier, yes! I will refactor the script accordingly.

Reply:
I implemented this accordingly. This is so much better!